ledger/streaming/client/src/scp_validator.rs

```diff
@@ -0,0 +1,343 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+//! Performs SCP validation on streams from multiple peers and merges them into
+//! a single stream
+
+use crate::streaming_futures::{Ready, Ready::NotReady};
+use futures::{stream, Stream, StreamExt};
+use mc_common::{
+    logger::{log, Logger},
+    NodeID,
+};
+use mc_consensus_scp::{GenericNodeId, QuorumSet, QuorumSetMember, SlotIndex};
+use mc_ledger_streaming_api::{BlockStream, BlockStreamComponents, Result as StreamResult};
+use mc_transaction_core::BlockID;
+use std::{collections::HashMap, future};
```

Comment:
I don't think that needs to be used. mc_common:HashMap has uses a specific cryptorng generator and limited architecture targets (x86) to play nicely with the enclave. This is working with public info, so it doesn't need those modifications. If it's thought that hashbrown would be faster then we should Hashbrown library directly since it uses the faster default Ahash (MC_common uses siphash).